modules/gitops-argo-cd-dynamic-scaling.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-argo-cd-dynamic-scaling_{context}"]
+= Enabling dynamic scaling of shards of the Argo CD Application Controller
```

Comment:
Hello @masauso-lungu as per [RHDEVDOCS-5856](https://issues.redhat.com/browse/RHDEVDOCS-5856), this feature is TP so we need to include the TP snippet before approving it, thank you!

Reply:
Thanks, @ekristova for posting the comment as per our discussion. +1 to the comment.

Reply:
Noted!, Thank you @ekristova.

Reply:
Addressed!